src/rules/selector-type-case/index.js

```diff
@@ -0,0 +1,66 @@
+import selectorParser from "postcss-selector-parser"
+import {
+  cssRuleHasSelectorEndingWithColon,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "selector-type-case"
+
+export const messages = ruleMessages(ruleName, {
+  expected: (h, v) => `Expected "${h}" to be "${v}"`,
```

Comment:
I'm not sure what h and v stand for. Can you go with something like "actual' and "expected" instead?